impeller/display_list/skia_conversions.cc

```diff
@@ -57,6 +57,7 @@ Path ToPath(const SkPath& path, Point shift) {
 
   PathBuilder builder;
   PathData data;
+  builder.Reserve(path.countPoints() + 16, path.countVerbs() + 16);
```

Comment:
since our storage more closely aligns with the SkPath, we can guestimate more easily...

Reply:
Added a line here